### PR TITLE
Fix bazel integration

### PR DESCRIPTION
**Pull request by alan-agius4**
//cc @alexeagle

### CODE REVIEW

File: packages/angular_devkit/build_angular/src/utils/process-bundle.ts [hunk 1]
```diff
@@ -128,7 +128,13 @@ export async function process(options: ProcessBundleOptions): Promise<ProcessBun
       babelrc: false,
       // modules aren't needed since the bundles use webpack's custom module loading
       // 'transform-typeof-symbol' generates slower code
-      presets: [['@babel/preset-env', { modules: false, exclude: ['transform-typeof-symbol'] }]],
+      presets: [[
+        require.resolve('@babel/preset-env'),
+        {
+          modules: false,
```
Comment (clydin) [inline packages/angular_devkit/build_angular/src/utils/process-bundle.ts]: Can you move the relevant comments next to each option as well?

File: packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts [hunk 2]
```diff
@@ -489,6 +489,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
       alias,
     },
     resolveLoader: {
+      symlinks: !buildOptions.preserveSymlinks,
```
Comment (clydin) [inline packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts]: Related: We should check to make sure all loaders are resolved  (`require.resolve`) before use to ensure the proper logic is used.  Especially relevant when peer dependencies are involved but also helpful for eventual pnp support.

File: packages/angular_devkit/build_angular/src/utils/process-bundle.ts (hunk 1 above)
Comment (alan-agius4) [reply packages/angular_devkit/build_angular/src/utils/process-bundle.ts]: Moved

**Comment by alexeagle** [comment]
I think this PR should no longer be necessary once https://github.com/bazelbuild/rules_nodejs/pull/1430 lands
It will cause us to lay out the normal `node_modules` tree in the working directory and patch the symlink primitives in node, when you call architect via a load from index.bzl

**Comment by mrmeku** [comment]
@alexeagle That might not necessarily be the case until we change the generated `nodejs_test` target to have its entry point live from within the normal `node_modules` directory. As it is the entry point will still be `external/npm` after that PR lands.

**Pull request merged**